uiService/zkVersion.go

```diff
@@ -182,7 +170,7 @@ func (zks *zkVersionStore) updateLocalCurrentVersion(version UIVersion) {
 }
 
 func (zks *zkVersionStore) getVersionFromZK() (UIVersion, error) {
-	data, err := zks.client.Get(zks.versionPath)
+	data, _, err := zks.client.Get(zks.versionPath)
 	if err != nil {
 		return UIVersion(""), errors.Wrap(err, "unable to get version from zk")
```

Comment:
What is the advantage of `type UIVersion string` and why don't we use it throughout?